visallo/visallo
# Upgrade Visallo to Elasticsearch 5.5 & log4j2

As a result of upgrading to Elasticsearch 5.5, this PR also upgrades Visallo to log4j2.


Testing Instructions:
1.) Migrate an existing Visallo installation from Elasticsearch 1.7 to Elasticsearch 5.5
2.) Verify that all expected data is present
3.) Verify search functionality

CHANGELOG
Changed: Upgraded Elasticsearch from version 1.7 to version 5.5
Changed: Upgraded log4j to log4j2
Changed: Term aggregations are no longer supported for properties that are indexed as FULL_TEXT only.
